pkgs/development/web/protege-distribution/default.nix

```diff
@@ -56,7 +58,7 @@ stdenv.mkDerivation rec {
     (makeDesktopItem {
       name = "Protege";
       desktopName = "Protege Desktop";
-      icon = "protege.ico";
+      icon = "protege";
```

Comment:
I committed the StartupWMClass modification, but my graphical environment doesn't pick the additional config up (probably because of its non-compliance). Please, let me know if it looks correct on your end.

Reply:
I think you need to logout of your graphical session and log back in again, that updates the menu entries for me at least.

I installed the new version of protege and it was placed into the appropriate sub-menu.

However, I can not verify the window grouping will work correctly, as I don't have window grouping in xfce and I still get `> WM_CLASS(STRING) = "sun-awt-X11-XFramePeer", "java-lang-Thread"` despite the `StartupWMClass` change.

According to [this SO answer](https://askubuntu.com/a/367851): _"So the field called StartupWMClass is a string that defines in which class the application is grouped in the Unity Launcher at startup."_
However, as seen [in this SO post](https://askubuntu.com/a/741940): _"StartupWMClass does not change the actual application window class, but only tells the desktop manager what window class should use to identify the application window(s)."_
On the other hand [this SO user claims](https://askubuntu.com/questions/653702/startupwmclass-will-not-change-wm-class-of-eclipse-ide#comment939567_653702): _"Same behavior here with my both eclipse installations (Java, C++). xprop WM_CLASS gives WM_CLASS(STRING) = "Eclipse", "Eclipse" in both cases [when using different StartupWMClasses]. But my running instances are not grouped."_

Alas, I'm quite puzzled now.

Reply:
Actually, I found the option for grouping applications. After conducting multiple tests, it does not work as expected. `StartupWMClass` has no effect on the grouping, at least in xfce, as the WM_Class still says "java-lang-Thread" - that's also the name of the group created.

The only proper solution I have found is to fix this upstream, in protege itself, e.g. see https://stackoverflow.com/a/29218320.

Unless someone else has a better idea, I'd suggest to remove that `StartupWMClass` entry, since it doesn't achieve anything apparently.
Sorry for bothering you with this, it seems I was missing a piece copying the desktop configuration from IntelliJ.

Reply:
Alright, removed and re-committed, anything else?